docs/zkapps/tutorials/08-custom-tokens.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: "Tutorial 8: Custom Tokens"
+hide_title: true
+sidebar_label: "Tutorial 8: Custom Tokens"
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Tutorial 8: Custom Tokens
+
+## Overview
+
+In this tutorial, we will discuss how to create custom tokens.
+
+Mina comes with native support for custom tokens. Each account on Mina can also have tokens associated with it.
+
+To create a new token, one creates a smart contract, which becomes the manager for the token, and uses that contract to set the rules around how the token can be mint, burned, and sent.
+
+The manager account may also set a token symbol for its token.
+
+In this tutorial, we will review smart contract code that creates and manages new tokens. You can find a full example, including these smart contracts and code to launch and interact with them, [here](https://github.com/es92/zkApp-examples/tree/main/08-custom-tokens/src).
+
+You can also find a more extensive example, including all the ways to interact with token smart contracts, [here](https://github.com/o1-labs/snarkyjs/blob/main/src/lib/token.test.ts).
+
+## Basic Token Example
+
+To create a token manager smart contract, we will create a normal smart contract, whose methods call special functions that manipulate tokens.
```

Comment:
For the term "token manager", we have been using "token owner" in the docs already (see https://docs.minaprotocol.com/zkapps/advanced-snarkyjs/custom-tokens#token-owner). Would it make sense to rename "token manager" to "token owner" and link to that section or do you think that manager better conveys the message? Wondering what sounds more natural

Reply:
I would argue I think in favor of TokenManager, since users own the tokens too? But can see the idea behind Owner as well

Reply:
Token manager does feel a bit more intuitive